client/src/GamemasterInGame/Lobby.tsx

```diff
@@ -101,10 +105,10 @@ export function Lobby({ rolesList }: LobbyProps) {
                     gap="2"
                     style={{ overflowY: "auto" }}
                   >
-                    <GameMasterActions
-                      selectedRoles={characterSelectState.selectedRoles.value}
```

Comment:
Was showing twice, so removed the duplicate buttons.

Reply:
ah merge conflict